homework/hackyourtemperature/server.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+import express from 'express';
+import bodyParser from 'body-parser';
+
+let Server=express();
```

Comment:
```suggestion
const server=express();
```

If you do not change a variable, it should be a `const` so that nothing changes it. Also, convention is to lowercase instances but capitalize classes.

Example:
```javascript
class Server {}

const server = new Server();
```